tests/unit/test_config.py

```diff
@@ -402,3 +402,11 @@ def test_valid_config_file_with_redis() -> None:
     except Exception:
         print(traceback.format_exc())
         pytest.fail()
+
+
+def test_config_file_without_logging_config() -> None:
+    """Check how a configuration file without logging config is correctly."""
+    # when logging configuration is not provided, default values
+    # will be used
+    # it means the following call should not fail
+    config.init_config("tests/config/config_without_logging.yaml")
```

Comment:
I believe that happens in `tests/unit/test_models.py`.

Reply:
it does, but i don't know why you wouldn't check it here as well.  this is a more complete flow of the config loading.

test_models.py only confirms that if you initialize an empty logconfig, it gets info as the default.

this only confirms that if you load an empty config, it doesn't fail.

you don't have anything that proves the connection between the two (that if you load an empty config, it initializes an empty logconfig to the default)

i'm reasonably certain I could introduce a code change that would break the default loglevel behavior in a way that didn't break either of the two tests you have here.

Reply:
the test description even says `Check how a configuration file without logging config is correctly initialized.`

but all it checks is that an exception isn't thrown.  It doesn't validate that the config is "correctly initialized"

Reply:
agree

Reply:
added @bparees